src/pytest_plugins/consume/hive_simulators/engine/test_via_engine.py

```diff
@@ -115,11 +116,15 @@ def test_blockchain_via_engine(
                                         f'got: "{payload_response.validation_error}" '
                                         f'expected: "{payload.validation_error}"'
                                     )
-                                    if client_strict_exception_matching:
+                                    if (
+                                        not client_strict_exception_matching
+                                        or not fork_strict_exception_matching
```

Comment:
I don't like having to repeat this `or` logic everywhere, case in point, line 104 in this file also requires this logic and was missed.
So I think we should create yet another fixture with client+fork strict exception matching.

Reply:
Thanks for catching that! And yes, this is much nicer / less error prone  :+1: